docs/test.md

```
@@ -25,13 +25,17 @@ but I've had trouble getting that to work.
-(probably from bosh.io) and name it `cpi-release.tgz`.
+that's going to be the bosh-cli source directory. For
+`bosh-docker-cpi-release`, download the docker cpi release (probably from
+bosh.io) and name it `cpi-release.tgz`. For `stemcell`, download a
+`bosh-warden-boshlite-ubuntu-noble` stemcell.
+
+The task brings up its own docker daemon by sourcing `start-docker`, so it has
+to run privileged (`-p`) on the `docker-cpi` image.
 
 Now we can fly execute:
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)